core/templates/css/oppia-material.css

```diff
@@ -2584,6 +2584,11 @@ th.mat-header-cell, td.mat-cell, td.mat-footer-cell {
 .mat-tab-group.mat-background-warn .mat-ripple-element, .mat-tab-nav-bar.mat-background-warn .mat-ripple-element {
   background-color: rgba(0, 0, 0, 0.12); }
 
+.mat-tab-header-pagination-controls-enabled .mat-tab-header-pagination {
```

Comment:
This is a generated CSS file. Can we find a different way to add this CSS using different selectors?

Reply:
@srijanreddy98 Thank you for the review. :)

Initially, I was looking for a way to avoid adding this CSS in the generated CSS file, but after a lot of digging, I could not find the selector for the arrow class, hence I added these lines here.

Once again, I will try to look for some other selectors that can do the same functionality, but it would be really helpful if you could point me to a certain direction where I need to look for them.

Reply:
Here are some of the problems with this fix:

It applies globally, this could cause a regression in places where this rule might not make sense to have.
Pixel values for CSS are not really appreciated currently. They maybe allowed only if relative units are not getting us a consistent experience.

Find the button in the component and add styles there. You can move this style to component where it used. That should scope it out from applying this changes to all other components. Try to avoid using the `.mat-tab-header-pagination-controls-enabled .mat-tab-header-pagination` selector for CSS. This not written by us and might break in the future if the material folks decide to change the class selectors.

You can try to use https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity to your advantage and get the desired results without having to use the material css selectors.

Reply:
Thanks :+1:  @srijanreddy98  for the feedback.

Got to learn so many amazing things :smile: 

Instead of adjusting the buttons in mat-tab group i thought of giving padding to the whole mat-tab-group this helps in maintaing consistency and avoids any problems for future changes.

here's a video showing the changes.

https://user-images.githubusercontent.com/96513964/198712809-8d8ee80a-8447-4d43-bae0-0d458ae4b826.mp4